forgerock-auth/src/main/java/org/forgerock/android/auth/callback/DeviceBindingCallback.kt

```diff
@@ -173,7 +173,7 @@ open class DeviceBindingCallback: AbstractCallback {
     /**
      * create the interface for the Authentication type(Biometric, Biometric_Fallback, none)
      */
-    open fun getDeviceBindInterface(): Authenticator {
+    protected open fun getAuthenticator(): Authenticator {
```

Comment:
This comment is not related to this PR, so no action item for now. But I think `Authenticator` is too generic for this interface. I would suggest rename it to something like `DeviceAuthenticator` or something more meaningful.

Reply:
yeah it is more generic, DeviceAuthenticator should work